prExplorer/API/Controllers/AccountController.cs

```diff
@@ -58,7 +58,38 @@ public async Task<ActionResult<UserDto>> Register(RegisterDto registerDto)
             {
                 DisplayName = registerDto.DisplayName,
                 UserName = registerDto.Username,
-                Email = registerDto.Email
+                Email = registerDto.Email,
+                Categories = new List<Category>
```

Comment:
I suggest storing the default categories somewhere in the database and only retrieving them here

Reply:
we can crate dummyUser and copy categories from that.